### PR TITLE
Add `.docx` documentation

### PR DESCRIPTION
**Pull request by MarkLindblad**

### CODE REVIEW

File: docs/source/aryn_cloud/aryn_partitioning_service.rst [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
-You can use the Aryn Partitioning Service to segment PDFs into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
+You can use the Aryn Partitioning Service to segment PDFs and Microsoft Office (.doc, .docx, .ppt, .pptx) documents and to automatically extract table data.  The service labels titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
```
Comment (jonfritz) [inline docs/source/aryn_cloud/aryn_partitioning_service.rst:4]: Office isn't used anymore. I'd replace this with "Microsoft Word (.docx and .dox), and Microsoft PowerPoint (.pptx and .ppt)"

File: docs/source/aryn_cloud/get_started.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## An Introduction to the Aryn Partitioning Service 
-You can use the Aryn Partitioning Service to easily chunk and extract data from complex PDFs. The Partitioning Service can extract paragraphs, tables and images. It returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment  PDF documents and extract a table and an image from those documents using the python aryn-sdk.
+You can use the Aryn Partitioning Service to easily chunk and extract data from complex documents. The Partitioning Service accepts PDF and Microsoft office formats (.docx, .doc, .pptx, .ppt). It can extract paragraphs, tables and images. It returns detailed information about the components it has just identified in a JSON object.  The following two sections will walk you through two examples where we segment PDF documents and extract a table and an image from those documents using the python aryn-sdk.
```
Comment (jonfritz) [inline docs/source/aryn_cloud/get_started.md:2]: Office isn't used anymore. I'd replace this with "Microsoft Word (.docx and .dox), and Microsoft PowerPoint (.pptx and .ppt)"

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst [hunk 3]
```diff
@@ -54,6 +54,25 @@ Helper Functions
     # Draw the detected bounding boxes on the pages. requires poppler
     images = draw_with_boxes("mydocument.pdf", data)
 
+Different Formats
```
Comment (jonfritz) [inline docs/source/aryn_cloud/accessing_the_partitioning_service.rst:57]: Different File Formats

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst [hunk 4]
```diff
@@ -54,6 +54,25 @@ Helper Functions
...
++++++++++++++++++
+
+It is easy to use multiple file types with aryn-sdk:
```
Comment (jonfritz) [inline docs/source/aryn_cloud/accessing_the_partitioning_service.rst:60]: It is easy to process files with different formats using the aryn-sdk:

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst [hunk 5]
```diff
@@ -107,4 +126,16 @@ Your results have been saved to ``document.json``.
...
+    cat document.json
+
+Different Formats
++++++++++++++++++
+
+.. code:: bash
+
+    export ARYN_API_KEY="PUT API KEY HERE"
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" | tee document.json
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.docx" | tee document.json
```
Comment (jonfritz) [inline docs/source/aryn_cloud/accessing_the_partitioning_service.rst:138]: "pdf=@..."  this is confusing for non PDFs. Can we change this?

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst [hunk 6]
```diff
@@ -107,4 +126,16 @@ Your results have been saved to ``document.json``.
 
 .. code:: bash
 
-    cat document.json
\ No newline at end of file
+    cat document.json
+
+Different Formats
```
Comment (jonfritz) [inline docs/source/aryn_cloud/accessing_the_partitioning_service.rst:131]: Different File Formats

**Review by jonfritz — CHANGES_REQUESTED** [review]
Thx for writing.

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst (hunk 5 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/accessing_the_partitioning_service.rst:138]: Yes. I will update this this week. @bsowell recommends we do not block the launch on this, unless you veto.

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst (hunk 6 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/accessing_the_partitioning_service.rst]: Done.

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst (hunk 4 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/accessing_the_partitioning_service.rst]: Done.

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst (hunk 3 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/accessing_the_partitioning_service.rst]: Done.

File: docs/source/aryn_cloud/get_started.md (hunk 2 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/get_started.md]: Done.

File: docs/source/aryn_cloud/aryn_partitioning_service.rst (hunk 1 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/aryn_partitioning_service.rst]: Done.

File: docs/source/aryn_cloud/aryn_partitioning_service.rst [hunk 7]
```diff
@@ -1,16 +1,18 @@
...
+You can use the Aryn Partitioning Service to segment PDFs, Microsoft Word (.docx and .dox) documents, and Microsoft PowerPoint (.pptx and .ppt) presentations and to automatically extract table data.  The service labels titles, tables, table rows and columns, images, and regular text. Bounding boxes are returned as JSON with their associated text for easy use. It leverages `Aryn's purpose-built AI model for document segmentation and labelling <https://huggingface.co/Aryn/deformable-detr>`_ that was trained using DocLayNet – an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
```
Comment (AbhijitP-009) [inline docs/source/aryn_cloud/aryn_partitioning_service.rst:4]: The first sentence sounds clunky.

Comment (AbhijitP-009) [reply docs/source/aryn_cloud/aryn_partitioning_service.rst:4]: How about "You can use the Aryn Partitioning Service to segment documents into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. The service can segment PDFs, Microsoft Word (.docx and .dox) documents, and Microsoft PowerPoint (.pptx and .ppt) file formats."

Comment (jonfritz) [reply docs/source/aryn_cloud/aryn_partitioning_service.rst:4]: +1

File: docs/source/aryn_cloud/accessing_the_partitioning_service.rst (hunk 5 above)
Comment (jonfritz) [reply docs/source/aryn_cloud/accessing_the_partitioning_service.rst:138]: Let's not block, but this should get updated in the next couple days if we don't block.

File: docs/source/aryn_cloud/aryn_partitioning_service.rst (hunk 7 above)
Comment (MarkLindblad) [reply docs/source/aryn_cloud/aryn_partitioning_service.rst]: Done.

**Review by jonfritz — APPROVED** [review]
LGTM!

**Pull request merged**